### PR TITLE
fix: added constraint

### PR DESCRIPTION
**Pull request by i-chvets**
Summary of changes:
- Added memory constraint to MySQL charm deployment.

NOTE: This is debug PR.

### CODE REVIEW

File: tests/integration/bundles/kfp_1.7_stable_install.yaml.j2 [hunk 1]
```diff
@@ -24,10 +24,9 @@ applications:
     trust: true
   kfp-db:
     charm: mysql-k8s
-    channel: 8.0/edge
+    channel: 8.0/stable
     scale: 1
-    options:
-      profile: testing
```
Comment (DnPlas) [inline tests/integration/bundles/kfp_1.7_stable_install.yaml.j2:30]: This option was suggested by the data team to have a constraint mysql.

Also, having `constraints: mem=250M` will just result in mysql not being able to even deploy.

Comment (i-chvets) [reply tests/integration/bundles/kfp_1.7_stable_install.yaml.j2:30]: Yes. I put it back in.

**Comment by i-chvets** [comment]
Looks like this time out did the trick.

**Comment by DnPlas** [comment]
I think we can close this PR, it seems like the CI in main does not have problems anymore.

**Comment by i-chvets** [comment]
This was a debug PR. Closing.

**Pull request closed without merging**